src/azure-cli/azure/cli/command_modules/synapse/tests/latest/recordings/test_sql_pool_classification_and_recommendation.yaml

```diff
@@ -14,15 +14,15 @@ interactions:
       - --name --workspace-name --resource-group --schema --table --column --label
         --information-type
       User-Agent:
-      - python/3.7.2 (Windows-10-10.0.18362-SP0) msrest/0.6.18 msrest_azure/0.6.3
-        azure-mgmt-security/0.4.1 Azure-SDK-For-Python AZURECLI/2.16.0
+      - python/3.7.4 (Windows-10-10.0.19041-SP0) msrest/0.6.21 msrest_azure/0.6.3
+        azure-mgmt-security/0.6.0 Azure-SDK-For-Python AZURECLI/2.24.0
       accept-language:
       - en-US
     method: GET
-    uri: https://management.azure.com/providers/Microsoft.Management/managementGroups/00000000-0000-0000-0000-000000000000/providers/Microsoft.Security/informationProtectionPolicies/effective?api-version=2017-08-01-preview
+    uri: https://management.azure.com/providers/Microsoft.Management/managementGroups/72f988bf-86f1-41af-91ab-2d7cd011db47/providers/Microsoft.Security/informationProtectionPolicies/effective?api-version=2017-08-01-preview
```

Comment:
It seems the recording file is wrong as usually it should be replaced by mocked value.